src/View/TwigView.php

```diff
@@ -77,6 +78,7 @@ class TwigView extends View
         'markdown' => [
             'engine' => null,
         ],
+        'deprecatedFeatures' => true,
```

Comment:
Perhaps we should default this to `false`.

Reply:
Seems odd to enable by default if we are going to have this setting. No one would switch it off.

@markstory Would you want this in your project?

Reply:
It would be good to have a warnings mode that I could turn on/off until I was finished upgrading. I'm not too concerned with what extensions are loaded and more that I can upgrade incrementally.

Reply:
There are no warnings for deprecated features at the moment. We could throw a warning when using the old-style tags, but not for using the old-style helpers.

Reply:
Having warnings on the old-style tags would be 💯 helpful. Old style helpers seem like low-cost to maintain for us, and could be kept around for a version to help with upgrading.